nupic/research/frameworks/pytorch/imagenet/mixins/log_every_loss.py

```diff
@@ -35,12 +35,14 @@ def setup_experiment(self, config):
         super().setup_experiment(config)
         self.error_loss_history = []
         self.complexity_loss_history = []
+        self.learning_rate_history = []
```

Comment:
There is already a LogEveryLearningRate mixin. I think it's better to keep these separate.

Reply:
Didn't realize, thanks. I'll revert the change.